### PR TITLE
Added Pump Event types for Manul Insulin Entry and the Other Markers

### PR DESCRIPTION
**Pull request by beached**
Added the Pump Event types for Manual Insulin and the Capture Event Other Markers

### CODE REVIEW

File: MinimedKit/PumpEventType.swift [hunk 1]
```diff
@@ -39,6 +39,8 @@ public enum PumpEventType: UInt8 {
     case BGReceived = 0x3f
     case JournalEntryMealMarker = 0x40
     case JournalEntryExerciseMarker = 0x41
+	case JournalEntryManualInsulinMarker = 0x42
```
Comment (loudnate) [inline MinimedKit/PumpEventType.swift:42]: Tab -> 4 spaces

File: RileyLink.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1388,34 +1394,35 @@
...
+						DevelopmentTeam = KZRVCA62XW;
 					};
 					C12EA251198B436800309FA4 = {
+						DevelopmentTeam = KZRVCA62XW;
 						TestTargetID = C12EA236198B436800309FA4;
 					};
 					C1B3830A1CD0665D00CE7782 = {
 						CreatedOnToolsVersion = 7.3;
-						DevelopmentTeam = UY678SP37Q;
+						DevelopmentTeam = KZRVCA62XW;
```
Comment (loudnate) [inline RileyLink.xcodeproj/project.pbxproj:1425]: Revert these?

File: RileyLink.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1881,6 +1890,7 @@
 				MTL_ENABLE_DEBUG_INFO = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = com.rileylink.RileyLinkBLEKit;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE = "";
```
Comment (loudnate) [inline RileyLink.xcodeproj/project.pbxproj:1893]: And these lines, too.

File: MinimedKit/PumpEventType.swift [hunk 4]
```diff
@@ -39,6 +39,8 @@ public enum PumpEventType: UInt8 {
...
+	case JournalEntryOther = 0x43
```
Comment (loudnate) [inline MinimedKit/PumpEventType.swift:43]: I believe this will conflict with #157

File: RileyLink.xcodeproj/project.pbxproj (hunk 2 above)
Comment (beached) [inline RileyLink.xcodeproj/project.pbxproj:1425]: Oops yah, that's the signing key.

File: RileyLink.xcodeproj/project.pbxproj (hunk 3 above)
Comment (beached) [inline RileyLink.xcodeproj/project.pbxproj:1893]: I assume so,

File: MinimedKit/PumpEventType.swift (hunk 4 above)
Comment (beached) [inline MinimedKit/PumpEventType.swift:43]: Only JournalEntryOther is 7

**Comment by ps2** [comment]
@beached I went ahead and got these records added as well.  Sorry for any duplication of efforts, and thanks for finding these!

https://github.com/ps2/rileylink_ios/pull/157
https://github.com/ps2/rileylink_ios/commit/91b552e3d0621c36d26ec732b2f360b0fdb1e75c

**Comment by beached** [comment]
@ps2 no worries

**Pull request closed without merging**